test/specs/EventsSpec.js

```diff
@@ -224,6 +224,16 @@ describe("Event Tests", function() {
 
     });
 
+    describe("Window Events", function() {
+      it("'resize' event is triggered properly and can be binded to", function() {
+        testSlider.on('resize', function() {
+          flag = true;
+        });
+        testSlider.data('slider')._resize();
```

Comment:
since the `resize()` method is not being publicly exposed, I would rather the test not invoke this method.

I saw your comment stating that the you cannot trigger the `resize` event on the global `window` object in the headless browser. Lets just go ahead and remove this test then